prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -337,4 +337,16 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Name"
 # HL7 Use
 - id: "hl7-use"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7-use"
\ No newline at end of file
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7-use"
+# Given Name
+- id: "cnn3-given-name"
```

Comment:
Looking at this a bit more, not sure this is possible. If we did it in a more generic way, then it would be hard to know exactly what to map going from FHIR to HL7v2 CNN segment.

Reply:
The different name cases are similar enough that we probably shouldn't have a `xcn5-suffix` and `cnn5-suffix` which both do the same thing, at least in cases where they’re this similar, it’s pretty clear that we could map both XCN5 and CNN5 from the same extension. They’ll probably both be handled through HumanName on Hl7->FHIR I think, so the same could probably happen in the other direction, though there might be some cases to figure out, for instance XCN.21 (professional suffix) doesn’t exist in CNN. Some of this depends on what gets merged first between this and the [HCI changes](https://github.com/CDCgov/prime-reportstream/pull/11850), but either way, I'd say we should get rid of the `cnn` in these extensions that won't be specific to CNN.

Reply:
I've been thinking on this and lean toward thinking we should keep these split out while mappings are still in progress; we are going to have some followup work to clean these up later and it might be best to leave decisions on what we can and can't consolidate until then.